Maddori.Apple/Maddori.Apple/Global/UIComponent/FeedbackTypeButtonView.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  FeedbackTypeButtonView.swift
+//  Maddori.Apple
+//
+//  Created by 김유나 on 2022/10/19.
+//
+
+import UIKit
+
+import SnapKit
+
+final class FeedbackTypeButtonView: UIButton {
+    private enum FeedbackType {
+        case continueType
+        case stopType
+    }
+    private enum Size {
+        static let buttonPadding: CGFloat = 11
+        static let width: CGFloat = 158
+        static let height: CGFloat = 46
+    }
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        render()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - property
+    
+    private var selectedType: FeedbackType = .continueType
+    private let continueShadowView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .white100
+        view.layer.shadowOpacity = 0.2
+        view.layer.shadowRadius = 2
+        view.layer.shadowOffset = CGSize(width: 0, height: 0)
+        view.layer.cornerRadius = 10
+        return view
+    }()
+    private let stopShadowView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .white100
+        view.layer.shadowOpacity = 0.2
+        view.layer.shadowRadius = 1
+        view.layer.shadowOffset = CGSize(width: 0, height: 0)
+        view.layer.cornerRadius = 10
+        return view
+    }()
+    private lazy var continueButton: UIButton = {
+        let button = UIButton()
+        button.setTitle("Continue", for: .normal)
+        button.setTitleColor(.white100, for: .normal)
+        button.titleLabel?.font = .main
+        button.backgroundColor = .blue200
+        button.clipsToBounds = true
+        button.layer.cornerRadius = 10
+        let buttonAction = UIAction { [weak self] _ in
+            self?.touchUpToSelectType(.continueType)
+        }
+        button.addAction(buttonAction, for: .touchUpInside)
```

Comment:
p4;

```suggestion
        let action = UIAction { [weak self] _ in
            self?.touchUpToSelectType(.continueType)
        }
        button.addAction(action, for: .touchUpInside)
```

요거도 욕심일 수 있겠지만, 최대한 한명이 짠 코드처럼 스타일을 맞추기 위해 `action`으로 변경해주시면 어떨까... 싶습니다.. ^______^

![image](https://user-images.githubusercontent.com/78677571/197186729-fb9203dd-1e2d-45ca-84d6-0a0a02158035.png)

Reply:
[68e685d](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/27/commits/68e685de40d10b3a26a9ce71d4f906e12b8adc8c)
수정해서 반영해두었습니다!
저 이모티콘은 어디서 구하신건지.. 귀엽네요

Reply:
직접 그렸다고 합니다 ㅎㅎ

Reply:
ㅋㅋㅋㅋㅋㅋㅋㅋㅋ 진짜요? 대박.. 저도 공유해주실 수 있나효..?